docs/app/js/app.js

```diff
@@ -286,6 +286,10 @@ function(SERVICES, COMPONENTS, DEMOS, PAGES, $location, $rootScope, $http, $wind
       pages: apiDocs.service.sort(sortByName),
       type: 'toggle'
     },{
+      name: 'Types',
```

Comment:
@petebacondarwin - IMO, we do not want a new 'Types' section (that will be shown in the Sidenav). What were your thoughts here ?